packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -204,102 +210,89 @@ const NewNetwork = decorate([
                   </div>
                 </Section>
                 <Section icon='info' title='newNetworkInfo'>
-                  {isPrivate ? (
-                    <div className='form-group'>
-                      <label>{_('newNetworkName')}</label>
-                      <input
-                        className='form-control'
-                        name='name'
-                        onChange={effects.linkState}
-                        required
-                        type='text'
-                        value={name}
-                      />
-                      <label>{_('newNetworkDescription')}</label>
-                      <input
-                        className='form-control'
-                        name='description'
-                        onChange={effects.linkState}
-                        type='text'
-                        value={description}
-                      />
-                      <label>{_('newNetworkEncapsulation')}</label>
-                      <Select
-                        className='form-control'
-                        name='encapsulation'
-                        onChange={effects.onChangeEncapsulation}
-                        options={[
-                          { label: 'GRE', value: 'gre' },
-                          { label: 'VxLAN', value: 'vxlan' },
-                        ]}
-                        value={encapsulation}
-                      />
-                    </div>
-                  ) : (
-                    <div className='form-group'>
-                      <label>{_('newNetworkInterface')}</label>
-                      <SelectPif
-                        multi={bonded}
-                        onChange={effects.onChangePif}
-                        predicate={pifPredicate}
-                        required={bonded}
-                        value={bonded ? pifs : pif}
-                      />
-                      <label>{_('newNetworkName')}</label>
-                      <input
-                        className='form-control'
-                        name='name'
-                        onChange={effects.linkState}
-                        required
-                        type='text'
-                        value={name}
-                      />
-                      <label>{_('newNetworkDescription')}</label>
-                      <input
-                        className='form-control'
-                        name='description'
-                        onChange={effects.linkState}
-                        type='text'
-                        value={description}
-                      />
```

Comment:
Why are you removing these?

Reply:
I'm not, I've refactor the code to avoid duplication.